test/commands/live/complete.test.ts

```diff
@@ -0,0 +1,17 @@
+import {expect, test} from '@oclif/test'
+
+describe('live:complete', () => {
+  test
+  .stdout()
+  .command(['live:complete'])
+  .it('runs hello', ctx => {
+    expect(ctx.stdout).to.contain('hello world')
+  })
+
+  test
+  .stdout()
+  .command(['live:complete', '--name', 'jeff'])
+  .it('runs hello --name jeff', ctx => {
+    expect(ctx.stdout).to.contain('hello jeff')
+  })
+})
```

Comment:
Are these placeholder tests or real?

Reply:
oclif generates tests when you make a command, you made the same one for `assets:create` :D

We should write actual tests at some point.